sklearn/tests/test_common.py

```diff
@@ -197,3 +202,23 @@ def test_get_params_invariance():
     for name, Estimator in estimators:
         if hasattr(Estimator, 'get_params'):
             yield check_get_params_invariance, name, Estimator
+
+
+def test_partial_fit():
```

Comment:
why did you do this in its own test? This means they are not part of ``check_estimator``, right?